.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,7 +1,5 @@
 name: Bug Report
 description: Create a report to help us improve
-labels:
```

Comment:
why remove this?

Reply:
based on last OPS discussion, I was under the impression that we'd like to have two distinct status: uncategorized/no-labels and `pending-triage`

Reply:
ah gotcha